datanode/sqlstore/migrations/0059_add_quantum_amount_in_rewards.sql

```diff
@@ -0,0 +1,25 @@
+-- +goose Up
+
+alter table rewards
+    add column if not exists quantum_amount HUGEINT null;
+
+-- +goose StatementBegin
+-- This computes the quantum_amount for old data.
+do
+$$
+    begin
+        update rewards
+        set quantum_amount = (select assets.quantum * rewards.amount from assets where assets.id = rewards.asset_id);
```

Comment:
would it not be better from assets_current?

Reply:
Didn't notice that view. Will do